### ISSUE TITLE
HC-App: Name sorting in Hash Cash view inconsistent with other screens

### ISSUE THREAD

**Issue by diabWH3** [James-A-White/HarrierCentral#267]
On the "Hash Cash" admin page, I'm seeing similar names sorted incorrectly (out of order, alphanumerically). Specifically "Dick Shank Redemption" coming before "Dick in a Box". These names show in correct order in the "Manual check in" screen and other places within the app.

For an example, here's info from the recent SeaMon H3 #333 trail:
![Screenshot_20220216-104248](https://user-images.githubusercontent.com/49632769/154334597-d9046880-7bb3-4074-92bb-0f2d85b1a96d.png)
![Screenshot_20220216-103328](https://user-images.githubusercontent.com/49632769/154334600-fbd26e0c-a24f-4db6-9a62-1b8e80acd4dc.png)

**Comment by James-A-White** [comment]
Good catch!  I think it has to do with me not handling case correctly. I should force all to lower case before sorting. Many thanks! 

I've been spending the last month or so focusing on the Web portal (and upgrading my development environment to a new machine... been harder than I thought as I migrated to an Apple M1 ARM Mac). Shortly I'll move my focus back to the app and sweep up a lot of these small issues. Thanks!
